pkg/controllers/management/auth/role_template_handler.go

```diff
@@ -0,0 +1,65 @@
+package auth
+
+import (
+	v3 "github.com/rancher/types/apis/management.cattle.io/v3"
+	"github.com/rancher/types/config"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/tools/cache"
+)
+
+const (
+	roleTemplateController  = "mgmt-auth-role-template-controller"
```

Comment:
the word handler would be more appropriate than controller (i realize previous examples contradict this). A controller is a k8s construct. a hander is a norman construct. 1 controller can execute (via the norman framework) many handlers